test/topology_custom/test_table_desc_read_barrier.py

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import logging
+import time
+import pytest
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.rest_client import inject_error
+from test.pylib.util import wait_for_cql_and_get_hosts
+from test.topology.conftest import skip_mode
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+@skip_mode('release', 'error injections are not supported in release mode')
+async def test_table_desc_read_barrier(request, manager: ManagerClient) -> None:
+    """
+    Regression test for #19213.
+
+    Test that the schema is always updated after read barrier.
+
+    Arrange:
+        - start 2 nodes
+        - create a table
+        - inject error (sleep) on the first node
```

Comment:
Maybe we call this function inject_error(), but you are injecting a sleep, not an error.
It can be useful to say in a couple of words *where* in the server code you inject this sleep (before completing something?).

Reply:
improved the comment